### DISCUSSION TITLE
Access to Github with GithubApps (ID and privatekey) using c#/.net

### DISCUSSION THREAD

**Discussion by fmouraine** [octokit/.github#66] [Q&A]
Hello,

I see that with node I can use :
import { createAppAuth } from "@octokit/auth-app"
(like in https://github.com/tibdex/github-app-token/blob/main/src/create-installation-access-token.ts)
but I do not see the equivalent in c#/.net
is it ongoing ?

Thanks
Regards,

François

**Comment by gr2m** [comment]
I'm not sure if there is a built-in feauter in `ocotkit.net` like we have it in the JavaScript Octokit, but here are some docs that might help

- https://github.com/octokit/handbook#github-app-authentication-json-web-token
- https://github.com/octokit/handbook#github-app-authentication-json-web-token
